datadog_checks_base/datadog_checks/base/stubs/datadog_agent.py

```diff
@@ -90,6 +93,15 @@ def set_hostname(self, hostname):
     def reset_hostname(self):
         self._hostname = 'stubbed.hostname'
 
+    def get_host_tags(self):
+        return self._host_tags
+
+    def set_host_tags(self, tagsDict):
+        self._host_tags = json.dumps(tagsDict)
+
+    def reset_host_tags(self):
+        self._host_tags = "{}"
```

Comment:
nit: I see this is only used for tests. Since it's not actually an [exposed method](https://github.com/DataDog/datadog-agent/pull/28027/files) other than the stub, let's make it private?

Reply:
yup good catch. making this and set_host_tags private 👍